modules/ROOT/pages/release-notes.adoc

```diff
@@ -369,3 +369,11 @@ A standalone docker image that package the LDAP Synchronizer was released under
 * BPO-664 - Security risk: API/formsDocumentImage executes injected script
 * BR-454 - Sometimes CallActivity are left Completed without target process
 * RUNTIME-50/BR-621 - All LDAP group objectclass checks are now case-insensitive.
+
+== Known issues
```

Comment:
== image:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
With the addition of the troubleshooting icon in the 2021.1 folder.

Reply:
For me, there is a semantic difference between both.